packages/client/src/hooks/use-balance.ts

```diff
@@ -0,0 +1,167 @@
+import { useState, useEffect } from 'react';
+import { ethers } from 'ethers';
+import erc20Abi from 'constants/abis/erc20.json';
+import { Wallet, WalletBalances } from 'types/states';
+import { UniswapPair } from '@sommelier/shared-types';
+const EXCHANGE_ADD_ABI_ADDRESS = '0xFd8A61F94604aeD5977B31930b48f1a94ff3a195';
```

Comment:
Probably good to move these to config files at this point, especially because they might be different on different networks

Reply:
sounds good. ill merge this and follow up with another PR that includes approval and adding liquidity, can move it to a config in that PR.